ROSCO_toolbox/inputs/toolbox_schema.yaml

```diff
@@ -864,6 +870,44 @@ properties:
                             type: number
                         description: Pitch angle offsets for each blade (array with length of 3)
                         units: rad
+                    AWC_Mode:           
+                        type: number
+                        minimum: 0
+                        maximum: 1
+                        default: 0
+                        description: Active wake control mode {0 - not used, 1 - SNL method}
+                    AWC_NumModes:
+                        type: number
+                        description: Number of AWC modes
+                        units: rad
+                        default: 1
+                    AWC_n:
+                        type: array
+                        items:
+                            type: number
+                        description: AWC azimuthal number
+                        default: [1]
+                    AWC_omega:
+                        type: array
+                        items:
+                            type: number
+                        description: AWC frequency [rad/s]
+                        units: rad/s
+                        default: [0.314159]
+                    AWC_amp:
+                        type: array
+                        items:
+                            type: number
+                        description: AWC amplitude [rad]
+                        units: rad/s
```

Comment:
Units should be rad not rad/s

Reply:
And, personally, I would change the units here to degrees to make it more intuitive, but that also requires a (small) change in the controller.